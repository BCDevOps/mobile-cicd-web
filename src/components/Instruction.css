@font-face {
  font-family: 'MyriadWebPro';
  src: url('/font/MyriadWebPro.ttf') format('truetype');
}

.instructions .toggle {
  height: 20px;
}

.instructions h3 {
  font-size: 20px;
  color: #00316b;
  margin: 1.5em 0 1em;
  border-bottom: 1px solid #ddd;
  padding-bottom: 0.3em;
}

.instructions .instruction {
  max-height: 20px;
  overflow: hidden;
}

.instructions .instruction.on {
  max-height: 100000px;
}

.instructions ol {
  counter-reset: li;
  list-style: none;
  position: relative;
  padding-bottom: 10px;
}

.instructions ol li {
  padding: 5px 0 5px 60px;
  position: relative;
  margin-bottom: 20px;
}

.instructions ol li::before {
  content: counter(li);
  counter-increment: li;
  position: absolute;
  top: 0;
  left: 0;
  height: 100%;
  width: 30px;
  padding: 0 10px 0 0;
  color: #999;
  font-size: 22px;
  font-weight: 700;
  line-height: 35px;
  text-align: right;
  border-right: 1px solid #ddd;
}

.instructions p {
  margin: 1em 0;
}

.instructions pre {
  padding: 8px 10px;
  margin: 10px 0;
  border: 1px solid #ddd;
  border-bottom-width: 2px;
  -webkit-border-radius: 3px;
  -moz-border-radius: 3px;
  border-radius: 3px;
  -moz-background-clip: padding;
  -webkit-background-clip: padding-box;
  background-clip: padding-box;
  -webkit-font-smoothing: inherit;
  overflow: auto;
}
