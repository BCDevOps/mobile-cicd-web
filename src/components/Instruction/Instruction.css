.instructions h3 {
  font-size: 20px;
  color: #00316b;
  margin-bottom: 1em;
  border-bottom: 1px solid #ddd;
  padding-bottom: 0.3em;
}

.instructions ol {
  counter-reset: li;
  list-style: none;
  position: relative;
  padding-bottom: 10px;
}

.instructions ol li {
  padding: 2px 55px;
  position: relative;
}

.instructions ol li::before {
  content: counter(li);
  counter-increment: li;
  position: absolute;
  left: 0;
  height: 100%;
  width: 36px;
  color: #999;
  font-size: 22px;
  border-right: 1px solid #ddd;
}

.instructions p {
  margin: 0.1em 0 2em 0;
}

.instructions pre {
  color: white;
  background-color: #494848;
}
