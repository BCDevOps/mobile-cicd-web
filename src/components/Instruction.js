import React, { Component } from 'react';
import './Instruction.css';
import { XML_SAMPLES } from '../constants';
import Toggle from './Toggle';
import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';
import { tomorrow } from 'react-syntax-highlighter/dist/styles/prism';

class Instruction extends Component {
  render() {
    return (
      <div className="instructions">
        <h3>Steps to Upload Your App</h3>
        <Toggle title="XcodeArchive">
          <p>To package up an xcarchive to submit for signing you need to:</p>
          <ol>
            <li>
              <p>Create a folder to hold the xcarchive and options.plist</p>
            </li>
            <li>
              <p>Copy the xcarchive from xcode into the folder from step 1</p>
            </li>
            <li>
              <p>
                Create or copy your options.plist from step 1, you could update the content from below:
              </p>
              <SyntaxHighlighter language="xml" style={tomorrow}>
                {XML_SAMPLES.PLIST}
              </SyntaxHighlighter>
            </li>
          </ol>
        </Toggle>
        <Toggle title="IPA">
          <p>To package up an xcarchive to submit for signing you need to:</p>
          <ol>
            <li>
              <p>Create a folder to hold the xcarchive and options.plist</p>
            </li>
            <li>
              <p>Copy the xcarchive from xcode into the folder from step 1</p>
            </li>
            <li>
              <p>
                Create or copy your options.plist from step 1, you could update the content from below:
              </p>
            </li>
          </ol>
        </Toggle>
        <Toggle title="APK">
          <p>To package up an xcarchive to submit for signing you need to:</p>
          <ol>
            <li>
              <p>Create a folder to hold the xcarchive and options.plist</p>
            </li>
            <li>
              <p>Copy the xcarchive from xcode into the folder from step 1</p>
            </li>
            <li>
              <p>
                Create or copy your options.plist from step 1, you could update the content from below:
              </p>
            </li>
          </ol>
        </Toggle>
      </div>
    );
  }
}

export default Instruction;
