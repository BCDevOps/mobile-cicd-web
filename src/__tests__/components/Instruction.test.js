import React from 'react';
import { shallow } from 'enzyme';
import Instruction from '../../components/Instruction';

describe('Instruction Component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Instruction />);
    expect(wrapper).toMatchSnapshot();
  });
});
