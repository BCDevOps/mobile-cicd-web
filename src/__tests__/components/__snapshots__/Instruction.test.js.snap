// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instruction Component matches snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Instruction />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3>
          Steps to Upload Your App
        </h3>,
        <Toggle
          title="XcodeArchive"
        >
          <p>
            To package up an xcarchive to submit for signing you need to:
          </p>
          <ol>
            <li>
              <p>
                Create a folder to hold the xcarchive and options.plist
              </p>
            </li>
            <li>
              <p>
                Copy the xcarchive from xcode into the folder from step 1
              </p>
            </li>
            <li>
              <p>
                Create or copy your options.plist from step 1, you could update the content from sample below:
              </p>
              <pre>
                
  &lt;?xml version="1.0" encoding="UTF-8"?&gt;
  &lt;!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd"&gt;
  &lt;plist version="1.0"&gt;
    &lt;dict&gt;
      &lt;key&gt;method&lt;/key&gt;
      &lt;string&gt;app-store&lt;/string&gt;
      &lt;key&gt;signingStyle&lt;/key&gt;
      &lt;string&gt;automatic&lt;/string&gt;
      &lt;key&gt;stripSwiftSymbols&lt;/key&gt;
      &lt;true/&gt;
      &lt;key&gt;uploadSymbols&lt;/key&gt;
      &lt;true/&gt;
    &lt;/dict&gt;
  &lt;/plist&gt;
  
              </pre>
            </li>
            <li>
              <p>
                ZIP up the folder for submission
              </p>
            </li>
          </ol>
        </Toggle>,
        <Toggle
          title="IPA"
        >
          <p>
            Signing IPA files is currently Alpha. Please use xcarchive.
          </p>
        </Toggle>,
        <Toggle
          title="APK"
        >
          <p>
            To package up an android package to submit for signing you need to:
          </p>
          <ol>
            <li>
              <p>
                Select RELEASE mode in the Build Variant, and build APK
              </p>
            </li>
            <li>
              <p>
                Drag and drop the apk
              </p>
            </li>
          </ol>
        </Toggle>,
      ],
      "className": "instructions",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Steps to Upload Your App",
        },
        "ref": null,
        "rendered": "Steps to Upload Your App",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <p>
              To package up an xcarchive to submit for signing you need to:
            </p>,
            <ol>
              <li>
                <p>
                  Create a folder to hold the xcarchive and options.plist
                </p>
              </li>
              <li>
                <p>
                  Copy the xcarchive from xcode into the folder from step 1
                </p>
              </li>
              <li>
                <p>
                  Create or copy your options.plist from step 1, you could update the content from sample below:
                </p>
                <pre>
                  
  &lt;?xml version="1.0" encoding="UTF-8"?&gt;
  &lt;!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd"&gt;
  &lt;plist version="1.0"&gt;
    &lt;dict&gt;
      &lt;key&gt;method&lt;/key&gt;
      &lt;string&gt;app-store&lt;/string&gt;
      &lt;key&gt;signingStyle&lt;/key&gt;
      &lt;string&gt;automatic&lt;/string&gt;
      &lt;key&gt;stripSwiftSymbols&lt;/key&gt;
      &lt;true/&gt;
      &lt;key&gt;uploadSymbols&lt;/key&gt;
      &lt;true/&gt;
    &lt;/dict&gt;
  &lt;/plist&gt;
  
                </pre>
              </li>
              <li>
                <p>
                  ZIP up the folder for submission
                </p>
              </li>
            </ol>,
          ],
          "title": "XcodeArchive",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "To package up an xcarchive to submit for signing you need to:",
            },
            "ref": null,
            "rendered": "To package up an xcarchive to submit for signing you need to:",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <li>
                  <p>
                    Create a folder to hold the xcarchive and options.plist
                  </p>
                </li>,
                <li>
                  <p>
                    Copy the xcarchive from xcode into the folder from step 1
                  </p>
                </li>,
                <li>
                  <p>
                    Create or copy your options.plist from step 1, you could update the content from sample below:
                  </p>
                  <pre>
                    
  &lt;?xml version="1.0" encoding="UTF-8"?&gt;
  &lt;!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd"&gt;
  &lt;plist version="1.0"&gt;
    &lt;dict&gt;
      &lt;key&gt;method&lt;/key&gt;
      &lt;string&gt;app-store&lt;/string&gt;
      &lt;key&gt;signingStyle&lt;/key&gt;
      &lt;string&gt;automatic&lt;/string&gt;
      &lt;key&gt;stripSwiftSymbols&lt;/key&gt;
      &lt;true/&gt;
      &lt;key&gt;uploadSymbols&lt;/key&gt;
      &lt;true/&gt;
    &lt;/dict&gt;
  &lt;/plist&gt;
  
                  </pre>
                </li>,
                <li>
                  <p>
                    ZIP up the folder for submission
                  </p>
                </li>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <p>
                    Create a folder to hold the xcarchive and options.plist
                  </p>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Create a folder to hold the xcarchive and options.plist",
                  },
                  "ref": null,
                  "rendered": "Create a folder to hold the xcarchive and options.plist",
                  "type": "p",
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <p>
                    Copy the xcarchive from xcode into the folder from step 1
                  </p>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Copy the xcarchive from xcode into the folder from step 1",
                  },
                  "ref": null,
                  "rendered": "Copy the xcarchive from xcode into the folder from step 1",
                  "type": "p",
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <p>
                      Create or copy your options.plist from step 1, you could update the content from sample below:
                    </p>,
                    <pre>
                      
  &lt;?xml version="1.0" encoding="UTF-8"?&gt;
  &lt;!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd"&gt;
  &lt;plist version="1.0"&gt;
    &lt;dict&gt;
      &lt;key&gt;method&lt;/key&gt;
      &lt;string&gt;app-store&lt;/string&gt;
      &lt;key&gt;signingStyle&lt;/key&gt;
      &lt;string&gt;automatic&lt;/string&gt;
      &lt;key&gt;stripSwiftSymbols&lt;/key&gt;
      &lt;true/&gt;
      &lt;key&gt;uploadSymbols&lt;/key&gt;
      &lt;true/&gt;
    &lt;/dict&gt;
  &lt;/plist&gt;
  
                    </pre>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Create or copy your options.plist from step 1, you could update the content from sample below:",
                    },
                    "ref": null,
                    "rendered": "Create or copy your options.plist from step 1, you could update the content from sample below:",
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "
  <?xml version=\\"1.0\\" encoding=\\"UTF-8\\"?>
  <!DOCTYPE plist PUBLIC \\"-//Apple//DTD PLIST 1.0//EN\\" \\"http://www.apple.com/DTDs/PropertyList-1.0.dtd\\">
  <plist version=\\"1.0\\">
    <dict>
      <key>method</key>
      <string>app-store</string>
      <key>signingStyle</key>
      <string>automatic</string>
      <key>stripSwiftSymbols</key>
      <true/>
      <key>uploadSymbols</key>
      <true/>
    </dict>
  </plist>
  ",
                    },
                    "ref": null,
                    "rendered": "
  <?xml version=\\"1.0\\" encoding=\\"UTF-8\\"?>
  <!DOCTYPE plist PUBLIC \\"-//Apple//DTD PLIST 1.0//EN\\" \\"http://www.apple.com/DTDs/PropertyList-1.0.dtd\\">
  <plist version=\\"1.0\\">
    <dict>
      <key>method</key>
      <string>app-store</string>
      <key>signingStyle</key>
      <string>automatic</string>
      <key>stripSwiftSymbols</key>
      <true/>
      <key>uploadSymbols</key>
      <true/>
    </dict>
  </plist>
  ",
                    "type": "pre",
                  },
                ],
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <p>
                    ZIP up the folder for submission
                  </p>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "ZIP up the folder for submission",
                  },
                  "ref": null,
                  "rendered": "ZIP up the folder for submission",
                  "type": "p",
                },
                "type": "li",
              },
            ],
            "type": "ol",
          },
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <p>
            Signing IPA files is currently Alpha. Please use xcarchive.
          </p>,
          "title": "IPA",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Signing IPA files is currently Alpha. Please use xcarchive.",
          },
          "ref": null,
          "rendered": "Signing IPA files is currently Alpha. Please use xcarchive.",
          "type": "p",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <p>
              To package up an android package to submit for signing you need to:
            </p>,
            <ol>
              <li>
                <p>
                  Select RELEASE mode in the Build Variant, and build APK
                </p>
              </li>
              <li>
                <p>
                  Drag and drop the apk
                </p>
              </li>
            </ol>,
          ],
          "title": "APK",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "To package up an android package to submit for signing you need to:",
            },
            "ref": null,
            "rendered": "To package up an android package to submit for signing you need to:",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <li>
                  <p>
                    Select RELEASE mode in the Build Variant, and build APK
                  </p>
                </li>,
                <li>
                  <p>
                    Drag and drop the apk
                  </p>
                </li>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <p>
                    Select RELEASE mode in the Build Variant, and build APK
                  </p>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Select RELEASE mode in the Build Variant, and build APK",
                  },
                  "ref": null,
                  "rendered": "Select RELEASE mode in the Build Variant, and build APK",
                  "type": "p",
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <p>
                    Drag and drop the apk
                  </p>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Drag and drop the apk",
                  },
                  "ref": null,
                  "rendered": "Drag and drop the apk",
                  "type": "p",
                },
                "type": "li",
              },
            ],
            "type": "ol",
          },
        ],
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            Steps to Upload Your App
          </h3>,
          <Toggle
            title="XcodeArchive"
          >
            <p>
              To package up an xcarchive to submit for signing you need to:
            </p>
            <ol>
              <li>
                <p>
                  Create a folder to hold the xcarchive and options.plist
                </p>
              </li>
              <li>
                <p>
                  Copy the xcarchive from xcode into the folder from step 1
                </p>
              </li>
              <li>
                <p>
                  Create or copy your options.plist from step 1, you could update the content from sample below:
                </p>
                <pre>
                  
  &lt;?xml version="1.0" encoding="UTF-8"?&gt;
  &lt;!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd"&gt;
  &lt;plist version="1.0"&gt;
    &lt;dict&gt;
      &lt;key&gt;method&lt;/key&gt;
      &lt;string&gt;app-store&lt;/string&gt;
      &lt;key&gt;signingStyle&lt;/key&gt;
      &lt;string&gt;automatic&lt;/string&gt;
      &lt;key&gt;stripSwiftSymbols&lt;/key&gt;
      &lt;true/&gt;
      &lt;key&gt;uploadSymbols&lt;/key&gt;
      &lt;true/&gt;
    &lt;/dict&gt;
  &lt;/plist&gt;
  
                </pre>
              </li>
              <li>
                <p>
                  ZIP up the folder for submission
                </p>
              </li>
            </ol>
          </Toggle>,
          <Toggle
            title="IPA"
          >
            <p>
              Signing IPA files is currently Alpha. Please use xcarchive.
            </p>
          </Toggle>,
          <Toggle
            title="APK"
          >
            <p>
              To package up an android package to submit for signing you need to:
            </p>
            <ol>
              <li>
                <p>
                  Select RELEASE mode in the Build Variant, and build APK
                </p>
              </li>
              <li>
                <p>
                  Drag and drop the apk
                </p>
              </li>
            </ol>
          </Toggle>,
        ],
        "className": "instructions",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Steps to Upload Your App",
          },
          "ref": null,
          "rendered": "Steps to Upload Your App",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <p>
                To package up an xcarchive to submit for signing you need to:
              </p>,
              <ol>
                <li>
                  <p>
                    Create a folder to hold the xcarchive and options.plist
                  </p>
                </li>
                <li>
                  <p>
                    Copy the xcarchive from xcode into the folder from step 1
                  </p>
                </li>
                <li>
                  <p>
                    Create or copy your options.plist from step 1, you could update the content from sample below:
                  </p>
                  <pre>
                    
  &lt;?xml version="1.0" encoding="UTF-8"?&gt;
  &lt;!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd"&gt;
  &lt;plist version="1.0"&gt;
    &lt;dict&gt;
      &lt;key&gt;method&lt;/key&gt;
      &lt;string&gt;app-store&lt;/string&gt;
      &lt;key&gt;signingStyle&lt;/key&gt;
      &lt;string&gt;automatic&lt;/string&gt;
      &lt;key&gt;stripSwiftSymbols&lt;/key&gt;
      &lt;true/&gt;
      &lt;key&gt;uploadSymbols&lt;/key&gt;
      &lt;true/&gt;
    &lt;/dict&gt;
  &lt;/plist&gt;
  
                  </pre>
                </li>
                <li>
                  <p>
                    ZIP up the folder for submission
                  </p>
                </li>
              </ol>,
            ],
            "title": "XcodeArchive",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "To package up an xcarchive to submit for signing you need to:",
              },
              "ref": null,
              "rendered": "To package up an xcarchive to submit for signing you need to:",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li>
                    <p>
                      Create a folder to hold the xcarchive and options.plist
                    </p>
                  </li>,
                  <li>
                    <p>
                      Copy the xcarchive from xcode into the folder from step 1
                    </p>
                  </li>,
                  <li>
                    <p>
                      Create or copy your options.plist from step 1, you could update the content from sample below:
                    </p>
                    <pre>
                      
  &lt;?xml version="1.0" encoding="UTF-8"?&gt;
  &lt;!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd"&gt;
  &lt;plist version="1.0"&gt;
    &lt;dict&gt;
      &lt;key&gt;method&lt;/key&gt;
      &lt;string&gt;app-store&lt;/string&gt;
      &lt;key&gt;signingStyle&lt;/key&gt;
      &lt;string&gt;automatic&lt;/string&gt;
      &lt;key&gt;stripSwiftSymbols&lt;/key&gt;
      &lt;true/&gt;
      &lt;key&gt;uploadSymbols&lt;/key&gt;
      &lt;true/&gt;
    &lt;/dict&gt;
  &lt;/plist&gt;
  
                    </pre>
                  </li>,
                  <li>
                    <p>
                      ZIP up the folder for submission
                    </p>
                  </li>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <p>
                      Create a folder to hold the xcarchive and options.plist
                    </p>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Create a folder to hold the xcarchive and options.plist",
                    },
                    "ref": null,
                    "rendered": "Create a folder to hold the xcarchive and options.plist",
                    "type": "p",
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <p>
                      Copy the xcarchive from xcode into the folder from step 1
                    </p>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Copy the xcarchive from xcode into the folder from step 1",
                    },
                    "ref": null,
                    "rendered": "Copy the xcarchive from xcode into the folder from step 1",
                    "type": "p",
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <p>
                        Create or copy your options.plist from step 1, you could update the content from sample below:
                      </p>,
                      <pre>
                        
  &lt;?xml version="1.0" encoding="UTF-8"?&gt;
  &lt;!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd"&gt;
  &lt;plist version="1.0"&gt;
    &lt;dict&gt;
      &lt;key&gt;method&lt;/key&gt;
      &lt;string&gt;app-store&lt;/string&gt;
      &lt;key&gt;signingStyle&lt;/key&gt;
      &lt;string&gt;automatic&lt;/string&gt;
      &lt;key&gt;stripSwiftSymbols&lt;/key&gt;
      &lt;true/&gt;
      &lt;key&gt;uploadSymbols&lt;/key&gt;
      &lt;true/&gt;
    &lt;/dict&gt;
  &lt;/plist&gt;
  
                      </pre>,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Create or copy your options.plist from step 1, you could update the content from sample below:",
                      },
                      "ref": null,
                      "rendered": "Create or copy your options.plist from step 1, you could update the content from sample below:",
                      "type": "p",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "
  <?xml version=\\"1.0\\" encoding=\\"UTF-8\\"?>
  <!DOCTYPE plist PUBLIC \\"-//Apple//DTD PLIST 1.0//EN\\" \\"http://www.apple.com/DTDs/PropertyList-1.0.dtd\\">
  <plist version=\\"1.0\\">
    <dict>
      <key>method</key>
      <string>app-store</string>
      <key>signingStyle</key>
      <string>automatic</string>
      <key>stripSwiftSymbols</key>
      <true/>
      <key>uploadSymbols</key>
      <true/>
    </dict>
  </plist>
  ",
                      },
                      "ref": null,
                      "rendered": "
  <?xml version=\\"1.0\\" encoding=\\"UTF-8\\"?>
  <!DOCTYPE plist PUBLIC \\"-//Apple//DTD PLIST 1.0//EN\\" \\"http://www.apple.com/DTDs/PropertyList-1.0.dtd\\">
  <plist version=\\"1.0\\">
    <dict>
      <key>method</key>
      <string>app-store</string>
      <key>signingStyle</key>
      <string>automatic</string>
      <key>stripSwiftSymbols</key>
      <true/>
      <key>uploadSymbols</key>
      <true/>
    </dict>
  </plist>
  ",
                      "type": "pre",
                    },
                  ],
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <p>
                      ZIP up the folder for submission
                    </p>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "ZIP up the folder for submission",
                    },
                    "ref": null,
                    "rendered": "ZIP up the folder for submission",
                    "type": "p",
                  },
                  "type": "li",
                },
              ],
              "type": "ol",
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <p>
              Signing IPA files is currently Alpha. Please use xcarchive.
            </p>,
            "title": "IPA",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Signing IPA files is currently Alpha. Please use xcarchive.",
            },
            "ref": null,
            "rendered": "Signing IPA files is currently Alpha. Please use xcarchive.",
            "type": "p",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <p>
                To package up an android package to submit for signing you need to:
              </p>,
              <ol>
                <li>
                  <p>
                    Select RELEASE mode in the Build Variant, and build APK
                  </p>
                </li>
                <li>
                  <p>
                    Drag and drop the apk
                  </p>
                </li>
              </ol>,
            ],
            "title": "APK",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "To package up an android package to submit for signing you need to:",
              },
              "ref": null,
              "rendered": "To package up an android package to submit for signing you need to:",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li>
                    <p>
                      Select RELEASE mode in the Build Variant, and build APK
                    </p>
                  </li>,
                  <li>
                    <p>
                      Drag and drop the apk
                    </p>
                  </li>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <p>
                      Select RELEASE mode in the Build Variant, and build APK
                    </p>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Select RELEASE mode in the Build Variant, and build APK",
                    },
                    "ref": null,
                    "rendered": "Select RELEASE mode in the Build Variant, and build APK",
                    "type": "p",
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <p>
                      Drag and drop the apk
                    </p>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Drag and drop the apk",
                    },
                    "ref": null,
                    "rendered": "Drag and drop the apk",
                    "type": "p",
                  },
                  "type": "li",
                },
              ],
              "type": "ol",
            },
          ],
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
